### PR TITLE
TST: add bigquery schema to datamgr

### LINKED ISSUE #2330: CI: Bigquery backend failing to load data
@tswast can you have a look at [this CI error](https://github.com/ibis-project/ibis/runs/984884965#step:4:1418) please?

Not sure what's the problem, I'd say that I'm calling the loading of data for bigquery in the same exact way as it was before.

### PR DESCRIPTION
**Pull request by tswast**
Closes #2330

### CODE REVIEW

File: ci/datamgr.py [hunk 1]
```diff
@@ -501,24 +506,17 @@ def bigquery(data_directory, ignore_missing_dependency, **params):
     except google.api_core.exceptions.Conflict:
         pass  # Skip if already created.
 
-    # Set up main data table.
+    # Set up main data tables.
     data_directory = Path(data_directory)
     functional_alltypes_path = data_directory / 'functional_alltypes.csv'
```
Comment (datapythonista) [inline ci/datamgr.py:511]: Minor thing, but I think it makes more sense that these two lines are moved after `# Load main data table.`. It's misleading that they are defined in this section, when they aren't used until later.

File: ci/datamgr.py [hunk 2]
```diff
@@ -532,7 +530,6 @@ def bigquery(data_directory, ignore_missing_dependency, **params):
     # Load an ingestion time partitioned table.
     functional_alltypes_path = data_directory / 'functional_alltypes.csv'
```
Comment (datapythonista) [inline ci/datamgr.py:531]: This is duplicated from above, isn't it?

File: ci/datamgr.py (hunk 1 above)
Comment (tswast) [reply ci/datamgr.py]: done in 04b773a40b5317b46b0a8aed939e488ac9f05213

File: ci/datamgr.py (hunk 2 above)
Comment (tswast) [reply ci/datamgr.py]: removed in https://github.com/ibis-project/ibis/commit/04b773a40b5317b46b0a8aed939e488ac9f05213

File: ci/datamgr.py [hunk 3]
```diff
@@ -477,9 +476,15 @@ def clickhouse(schema, tables, data_directory, **params):
 
 
 @cli.command()
+@click.option(
+    '-S',
+    '--schema',
+    type=click.File('rt'),
+    default=str(SCRIPT_DIR / 'schema' / 'bigquery.sql'),
```
Comment (jreback) [inline ci/datamgr.py:483]: can you add a help here (e.g. a schema is a sql file); is ``--schema`` common? e.g. its slightly misleading on what its doing

Comment (tswast) [reply ci/datamgr.py:488]: Yes, added.

It's not common to call schema a SQL file in my experience with BigQuery, but it's the same as the other backends are doing. I've added help everywhere this argument is defined.

**Comment by jreback** [comment]
thanks @tswast

**Pull request merged**